dashboard/db/migrate/20160718080224_remove_user_id_from_sections.rb

```diff
@@ -0,0 +1,9 @@
+class RemoveUserIdFromSections < ActiveRecord::Migration
+  def up
+    remove_column :sections, :user_id
+  end
+
+  def down
+    add_column :sections, :user_id, :integer
+  end
+end
```

Comment:
These migrations, if run consecutively, cause the `user_id` data in the `sections` table to be deleted. Is it possible to create a sequence of migrations, each in its own PR, that keep the DB in the same state if rolled back to any point?

For example, create the following sequential migrations (PRs) [these haven't been well thought]:
(1) UP: Create the SectionUsers table, populating it with data. DOWN: Drop the SectionsUsers table.
(2) UP: Drop the `user_id` column from `sections`. DOWN: Add the `user_id` column to `sections`. Populate it with data from the `SectionUsers` table.